src/tearDown.js

```diff
@@ -61,8 +62,9 @@ function handleUnmount (node) {
 
 export default function tearDown (node, part) {
   // bail out if node is reused. It might be on different index
```

Comment:
Change the comment now to 
```suggestion
  // bailout if the node is a non-renderable node or if the node is reused (It might be on the different index)
```